src/data/accessibility-quiz.ts

```diff
@@ -1,4 +1,41 @@
 const accessibilityQuiz = [
+  {
+    Question:
+      "When did World Wide Web Consortium first drafted standards for developing accessible websites?",
+    Answer: "1999",
+    Distractor1: "2001",
+    Distractor2: "1990",
+    Distractor3:
+      "1989",
+    Explanation:
+      "The World Wide Web Consortium first drafted standards for developing accessible websites back in 1999.",
+    Link: "https://www.freecodecamp.org/news/what-is-website-accessibility-18ce00ec990f/"
+  },
+  {
+    Question:
+      "What legal requirement has brought web accessibility to the forefront in recent years?",
+    Answer:"The Americans with Disabilities Act",
+    
+    Distractor1: "The Internet Freedom Act",
+    Distractor2: "The Web Accessibility Act",
+    Distractor3:
+      "The Digital Inclusion Law",
+    Explanation:
+"The Americans with Disabilities Act (ADA) has brought web accessibility to the forefront by requiring public spaces and commercial facilities to provide equal access to individuals with disabilities, which now includes web-based activities due to legal rulings.",
```

Comment:
fix formatting issue

```suggestion
    Explanation:
      "The Americans with Disabilities Act (ADA) has brought web accessibility to the forefront by requiring public spaces and commercial facilities to provide equal access to individuals with disabilities, which now includes web-based activities due to legal rulings.",

```